docker-compose.yml

```diff
@@ -107,6 +107,16 @@ services:
     image: arcblock/driver-did-abt
     ports:
       - "8101:8080"
+  driver-did-trustbloc:
+    container_name: driver-did-trustbloc
+    image: trustbloc/driver-did-trustbloc:0.1.3
+    environment:
+      - DID_METHOD_HOST_URL=0.0.0.0:8102
```

Comment:
@fqutishat If possible, can we also use environment variables here, and set the defaults in the .env file, as the other drivers do it? If you prefer we can also do this for you (@azuzi can help)..

Reply:
After merge I will do that

Reply:
@azuzi thank you so much